src/L1YearnEscrow.sol

```diff
@@ -120,10 +134,12 @@ contract L1YearnEscrow is L1Escrow {
     function _receiveTokens(
         uint256 amount
     ) internal virtual override whenNotPaused {
+        VaultStorage storage $ = _getVaultStorage();
+        require($.deposited + amount <= $.depositLimit, "deposit limit");
```

Comment:
I personally like to use string error codes when using requires (e.g. "DEPOSIT_LIMIT") - or custom errors with reverts.

Reply:
not sure i follow.

Are you saying just change the revert string to "DEPOSIT_LIMIT" instead of "deposit limit" ?

Reply:
yeah, but it's minor

I find it easier to integrate with frontends etc